src/cli.rs

```diff
@@ -168,6 +170,17 @@ pub fn build_args<'a, 'b>(version: &'b str) -> App<'a, 'b> {
                 .required(false)
                 .help("If specified, transactions are send in batches of specified size"),
         )
+        .arg(
+            Arg::with_name("keeper-authority")
```

Comment:
why do you want to call it authority instead of identity? I mean if I understand correctly you want to use to identify this connection as stacked by validator and hence need to provide node identity of an existing validator along with IP address of this validator (it might be different than this client IP). My understanding is that authority term is used for paying fees